Umbraco-Cloud/Troubleshooting/Deployments/Changes-Not-Being-Applied/index.md

```diff
@@ -0,0 +1,53 @@
+---
+versionFrom: 7.0.0
+---
+
+# Extraction error: Config transforms failing
+
+Sometimes you might notice that even though your deployments are coming through without errors, no changes are actually being applied on the Cloud environments.
+This might be due to Config Transforms failing during the extraction process.
+
+This issue may occur if you have added custom [Config Transforms](../../../Set-up/Config-Transforms) to your project.
+
+Since Config Transforms are applied on every data extraction, the behaviour will continue until the Config Transform files are either corrected, or removed.
+
+## How do I know I have this issue?
+
+This problem is a bit tricky to spot, as the only indicator is that changes you made on the original environment are not being applied even though the deployment was complete (although when pushing from local you might get a warning message).
```

Comment:
Consider using "source environment" and "target environment" here, instead of "original environment".
That will make it consistent with our other Cloud articles.